packages/xstate-wallet/src/workflows/approve-budget-and-fund.ts

```diff
@@ -219,7 +219,7 @@ const createLedger = (store: Store) => async (context: Initial): Promise<LedgerI
 
   const stateVars = {
     outcome: initialOutcome,
-    turnNum: bigNumberify(0),
+    turnNum: BigNumber.from(0),
```

Comment:
Can we use Zero from '@ethersproject/constants' for code like this? (basically +1 to @snario's comment)

Reply:
[21 files later](https://github.com/statechannels/monorepo/pull/1934/commits/0a24f385a509b09cb070beb23ea1e58bd2fb3f11) 😛